src/domain/reservation/manage/action/__tests__/ManageReservationsDropdown.test.js

```diff
@@ -5,11 +5,48 @@ import { UntranslatedManageReservationsDropdown as ManageReservationsDropdown }
 import { shallowWithIntl } from '../../../../../../app/utils/testUtils';
 import reservation from '../../../../../common/data/fixtures/reservation';
 
+const findButtonByLabel = (wrapper, label) => wrapper.find({ children: `ManageReservationsList.actionLabel.${label}` });
+
 describe('ManageReservationsDropdown', () => {
+  const defaultProps = {
+    reservation: reservation.build({ state: null }),
+    t: path => path,
+    userCanModify: true,
+  };
+  const getWrapper = props => shallowWithIntl(
+    <ManageReservationsDropdown {...defaultProps} {...props} />,
+  );
+
   test('renders correctly', () => {
-    const wrapper = shallowWithIntl(
-      <ManageReservationsDropdown reservation={reservation.build()} t={() => 'foo'} userCanModify />,
-    );
-    expect(toJSON(wrapper)).toMatchSnapshot();
+    expect(toJSON(getWrapper())).toMatchSnapshot();
+  });
+
+  test('show information, edit and cancel buttons', () => {
+    const wrapper = getWrapper();
+
+    expect(findButtonByLabel(wrapper, 'information').length).toEqual(1);
+    expect(findButtonByLabel(wrapper, 'edit').length).toEqual(1);
+  });
```

Comment:
Based on test description this is missing check for cancel button. Adding a test where `userCanCancel` is set to `true` would be benefitical, because ManageReservations is a resource for admins after all.

Reply:
Good catch. I fixed the description and added a test case for the cancel button 👍